src/EbmlString.cpp

```diff
@@ -64,10 +64,10 @@ EbmlString::EbmlString(const std::string & aDefaultValue)
 /*!
   \todo Cloning should be on the same exact type !
 */
-void EbmlString::SetDefaultValue(std::string & aValue)
+void EbmlString::SetDefaultValue(std::string aValue)
 {
   assert(!DefaultISset());
-  DefaultValue = aValue;
+  DefaultValue = std::move(aValue);
```

Comment:
This will not be necessary after changing the argument to `std::string const &`.

Reply:
right. that's the other way to do it.